app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -507,6 +507,39 @@ public boolean addWikidataEditTag(String revisionId) throws IOException {
         }).flatMapObservable(Observable::fromIterable);
     }
 
+    @Override
```

Comment:
Oops! Did this get added by mistake?

Reply:
yes sorry i was actually adding this code for #2392